### DISCUSSION TITLE
warning al ejecutar make

### DISCUSSION THREAD

**Discussion by GuilleStankeS** [IIC2133-PUC/2024-2#46] [Tarea 0] [answered]
Cuando ejecuto "make", se compila correctamente pero me aparecen mensajes como el siguiente:
clang: warning: -lm: 'linker' input unused [-Wunused-command-line-argument]

se que puedo evitarlo cambiando la linea 51 del makefile dejandolo así:
MTH=#-lm

está bien que haga esot? o dejo que aparezca el warning?

**Comment by sebacarrascop** [comment, marked as answer]
hola!, no pude replicar tu warning, incluso con el makefile del codigo base.
Sigues teniendo ese problema?

**Comment by waknaudt** [comment]
A mi me pasó en el pasado al tratar de compilar C en android. Si mal no recuerdo tenía algo que ver con los procesadores arm.

**Reply by tomastrivino** [reply]
jaja wow no cachaba que podías compilar C en android

**Reply by waknaudt** [reply]
Sep jajaja con la app termux puedes instalar clang y compilar desde la terminal de termux
